mupen64plus-video-parallel/upstream/gfx_m64p.c

```diff
@@ -309,6 +309,10 @@ EXPORT void CALL SetRenderingCallback(void (*callback)(int))
 
```

Comment:
For this plugin, make the pull request here:

https://github.com/mupen64plus-ae/parallel-rdp-standalone